src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
```

Comment:
Nice job using querySelector! That's what we typically use in real projects. But can you use a more descriptive name for the variable? You're targeting the root body of the HTML so something like `rootHtmlBody` or just `body` could work here